packages/react-openapi/package.json

```diff
@@ -6,7 +6,8 @@
             "types": "./dist/index.d.ts",
             "development": "./src/index.ts",
             "default": "./dist/index.js"
-        }
+        },
+        "./client": "./src/client.ts"
```

Comment:
Exporting the client component separately here because I found that If I add a client component (OpenAPIContextProvider) to the `/src/index` file it has the unfortunate side effect of turning all the server components exported in that file into client components.

Reply:
How does it work if you put `'use server'` at the top of the other file?

Reply:
Will that do anything? I thought `use server` was the default and `use client` is the only directive that really does anything, at the top of a file at least.

Reply:
Tried this and it doesn't work. Throws an error as can't export a client side component from a server component.